ignite/pkg/cosmosgen/generate_typescript.go

```diff
@@ -28,7 +48,41 @@ type generatePayload struct {
 }
 
 func newTSGenerator(g *generator) *tsGenerator {
-	return &tsGenerator{g}
+	tsg := &tsGenerator{g: g}
+	if _, err := exec.LookPath(protocGenTSProtoBin); err == nil {
+		tsg.isLocalProto = true
+	}
+
+	if !tsg.isLocalProto {
+		log.Printf("No '%s' binary found in PATH, using remote buf plugin for Typescript generation. %s", protocGenTSProtoBin, msgBufAuth)
```

Comment:
Can we pass the session event system or find a way to stop a spinner before it throws this message?

<img width="2434" height="190" alt="Screenshot 2025-08-04 at 16 02 35" src="https://github.com/user-attachments/assets/fb47b30a-ffc3-4bbd-a7b7-2f79c87bd3fb" />

Reply:
I thought about it too, but i thought it was best not to actually, as the whole cosmosgen package doesn't depend on the event bus / session from the `cliui` package.

Reply:
Sure! Let me try

Reply:
Sure! Let me try